### PR TITLE
Hotfix: remove annoying warning during job-name to job-ID resolution

### PR DESCRIPTION
**Pull request by ayushkovskiy**
Closes https://github.com/neuromation/platform-client-python/issues/1025

### CODE REVIEW

File: neuromation/cli/utils.py [hunk 1]
```diff
@@ -461,11 +461,6 @@ def volume_to_verbose_str(volume: Volume) -> str:
             f"{details} to a job-ID: {e}"
         )
     if jobs:
-        if len(jobs) > 1:
-            log.warning(
-                f"Found {len(jobs)} jobs matching {details}: "
-                ", ".join(job.id for job in jobs)
-            )
         job_id = jobs[-1].id
```
Comment (serhiy-storchaka) [inline neuromation/cli/utils.py]: Would not be better to return always *your* job if there is such in the found jobs and return the last job only if no of them are your?

Comment (serhiy-storchaka) [reply neuromation/cli/utils.py]: Ah, the owner is always specified.

Comment (ayushkovskiy) [reply neuromation/cli/utils.py]: Yes, as we agreed in https://github.com/neuromation/platform-client-python/issues/1024, `neuro kill job-name` will also pass the parameter `owner=myself` when resolving `job-name` to job-ID

**Comment by asvetlov** [comment]
Please make at least on Circle CI job green

**Comment by asvetlov** [comment]
Forgot to mention: please add changelog file.
The PR worth a release, I want to provide a text to users why the release is needed.

**Comment by ayushkovskiy** [comment]
> Forgot to mention: please add changelog file.

Thank you! added.

**Pull request merged**